src/RESTController.js

```diff
@@ -255,8 +256,12 @@ const RESTController = {
       }
 
       const payloadString = JSON.stringify(payload);
-      return RESTController.ajax(method, url, payloadString, {}, options).then(({ response }) => {
-        return response;
+      return RESTController.ajax(method, url, payloadString, {}, options).then(({ response, status })=>{
+        if (options.returnStatus) {
+          return { ...response, status };
```

Comment:
The only issue I see is that if the response has a status field it would get overridden.

`object.set('status', 'OK')` would no longer work.

perhaps  `return { ...response, _status: status };` and delete _status

Reply:
good catch and I like the solution too.